src/main/java/Task.java

```diff
@@ -0,0 +1,18 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
```

Comment:
I dont thinks the use of "this" for this.isDone is necessary and can be replaced with just isDone = false as isDone is not shadowed by anything